### PR TITLE
Create symmetry for pool functionalities

### PR DESCRIPTION
**Pull request by goncer**
xyk-pallet - Pool opeations: Simmetry
    ✕ GetBalance x-y and y-x pool (3534 ms)
    ✓ Minting x-y and y-x pool (32722 ms)
    ✓ Burning x-y and y-x pool (36122 ms)
    ✕ GetLiquidityAssetID x-y and y-x pool (15 ms)

@Stano, I can only see that GetLiquidityAssetId and pool symmetry is not fulfilled. Maybe we can meet with @gleb-urvanov  about what todo next.

UPDATE: 
After the discussion, it was accepted the symmetry won't be accomplished and SDK will do.
Adding into the tests that the symmetry is NOT for the GetBalance and GetLiquidityAssetId and later SKD replacement will do the job,
ready to be reviewed!

### CODE REVIEW

File: test/parallel/xyk-pallet.API.pool.test.ts [hunk 1]
```diff
@@ -434,3 +436,133 @@ describe("xyk-pallet - Pool tests: a pool can:", () => {
...
+
+    await waitNewBlock();
+    keyring = new Keyring({ type: "sr25519" });
+
+    // setup users
+    testUser1 = new User(keyring);
+
+    sudo = new User(keyring, sudoUserName);
+
+    //add two curerncies and balance to testUser:
```
Comment (EddyFi) [inline test/parallel/xyk-pallet.API.pool.test.ts:463]: Minor typo, `currencies`

File: test/parallel/xyk-pallet.API.pool.test.ts [hunk 2]
```diff
@@ -434,3 +436,133 @@ describe("xyk-pallet - Pool tests: a pool can:", () => {
...
+    [firstCurrency, secondCurrency] = await Assets.setupUserWithCurrencies(
```
Comment (EddyFi) [inline test/parallel/xyk-pallet.API.pool.test.ts:464]: I know this is a common pattern elsewhere, but would you be happy to use 'real' names in place of `firstCurrency` and `secondCurrency` such as BTC and ETH so it's easier to read?

File: test/parallel/xyk-pallet.API.pool.test.ts (hunk 1 above)
Comment (eddyjlsh) [inline test/parallel/xyk-pallet.API.pool.test.ts:463]: Minor typo: Currencies

**Review by eddyjlsh — APPROVED** [review]
LGTM, comment about naming users and currencies in Slack but not a blocker for this :)

File: test/parallel/xyk-pallet.API.pool.test.ts (hunk 1 above)
Comment (goncer) [reply test/parallel/xyk-pallet.API.pool.test.ts:463]: done

**Pull request merged**